trace/HTTP.md

```diff
@@ -0,0 +1,105 @@
+# HTTP
+
+This document explains tracing of HTTP requests with OpenCensus.
+
+## Spans
+
+Implementations MUST create a span for outgoing requests at the
+client and a span for incoming requests at the server.
+
+Span name is formatted as:
+
+* Sent.$host[:$port]/$path for outgoing requests.
```

Comment:
We use "Sent.className.methodName" in current java instrumentation. Should we change to follow this new convention? /cc @bogdandrutu

Reply:
@bogdandrutu agrees that we should use this naming format.

"Sent.className.methodName" should always be the same so it doesn't help identifying the span.

Reply:
Ack.